x-pack/solutions/security/plugins/security_solution/public/sourcerer/containers/use_init_sourcerer.tsx

```diff
@@ -370,4 +370,11 @@ export const useInitSourcerer = (
     signalIndexName,
     signalIndexNameSourcerer,
   ]);
+
+  return useMemo(
```

Comment:
return browser fields from `useInitSourcerer` as there was another call to `useSourcererDataView` in the home component that only returned browser fields...and we already made that call in the `useInit`, so now we just return it from there instead. 

Other change here:
https://github.com/elastic/kibana/pull/212482/files#diff-c46dd8c3c8b81dc65ae1d303e29c3c0e966fcd5e4588731828b46142b18d494aL42